provider/aws/releases.go

```diff
@@ -246,7 +246,7 @@ func (p *Provider) ReleasePromote(app, id string, opts structs.ReleasePromoteOpt
 			return err
 		}
 
-		ou, err := p.ObjectStore(app, "", bytes.NewReader(data), structs.ObjectStoreOptions{Public: options.Bool(true)})
+		ou, err := p.ObjectStore(app, "", bytes.NewReader(data), structs.ObjectStoreOptions{Presign: options.Bool(true)})
```

Comment:
We are storing the new templates that CF will use for the deployment so we create a pre-signed URL - and CloudFormation will be able to download them.